core/build.gradle.kts

```diff
@@ -1,6 +1,34 @@
 plugins {
     id("org.p2p.wallet.android.library")
     id("kotlin-kapt")
+    id("org.jetbrains.kotlinx.kover") version "0.7.2"
+}
+
+koverReport {
```

Comment:
it doesn't make sense to move these configs to buildSrc as every module has its own exclusion config etc